src/main/java/com/klarna/hiverunner/HiveShell.java

```diff
@@ -203,10 +203,10 @@ public interface HiveShell {
      */
     HiveConf getHiveConf();
 
-    void setCwd(Path cwd);
-
     Path getCwd();
 
+    void setCwd(Path cwd);
```

Comment:
Same here, I'l stop commenting now if you can please update this MR to just be reformatting the existing code and then I'll take another look. Let's get this MR sorted and then we can apply the same pattern of whatever we decide here to the other two.